src/components/IntegrationTest/IntegrationTestForm/IntegrationTestSection.tsx

```diff
@@ -23,13 +25,44 @@ const IntegrationTestSection: React.FC<Props> = ({ isInPage, edit }) => {
     type: 'input',
   });
 
+  const [, { value: ITSEnvName }] = useField<string>({
+    name: 'integrationTest.environmentName',
+    type: 'dropdown',
+  });
+  const [environments, environmentsLoaded] = useAllEnvironments();
```

Comment:
I am not sure if BYOC environment is supported for ephemeral environment. So you'll need to filter out the static environments for this dropdown.

Reply:
updated to filter out managed envs